src/stores/Create.ts

```diff
@@ -221,13 +254,69 @@ export const getInitialState = (
                 'Setting endpointSchema'
             );
         },
+        setResourceConfig: (key, value) => {
+            set(
+                produce((state) => {
+                    state.resourceConfig[key] =
+                        value ?? getDefaultJsonFormsData();
+                }),
+                false,
+                'Resource Config Changed'
+            );
+        },
+        getResourceConfigErrors: () => {
+            const { resourceConfig } = get();
+            let response: any[] = [];
+
+            if (Object.keys(resourceConfig).length > 0) {
+                map(resourceConfig, (config) => {
+                    const { errors } = config;
+                    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+                    if (errors && errors.length > 0) {
+                        response = response.concat(errors);
+                    }
+                });
+            } else {
+                // TODO (errors) Need to populate this object with something?
+                response = [{}];
+            }
+
+            return response;
+        },
```

Comment:
Might want to look into the `compute` middlewares that could make this better than just a plain function that needs to be re-run to get the most recent response.